docs/topics/db/queries.txt

```diff
@@ -361,6 +361,10 @@ every *second* object of the first 10::
 
     >>> Entry.objects.all()[:10:2]
 
+Filtering and ordering operations are not possible after slicing, as they would
+lead to inconsistent results across different database backends. For pagination
+purposes, filters, and sorting should be applied before slicing.
```

Comment:
I guess this means "filters and sorting" without a comma between them. (sorry my comment on the other PR was incorrect)